frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeSizeSection.tsx

```diff
@@ -1,27 +1,31 @@
 import * as React from 'react';
-import { FormGroup, FormSection, Select, SelectOption } from '@patternfly/react-core';
+import { FormGroup, FormSection, NumberInput, Select, SelectOption } from '@patternfly/react-core';
 import { UpdateObjectAtPropAndValue } from 'pages/projects/types';
 import { CreatingServingRuntimeObject, ServingRuntimeSize } from '../../types';
 import ServingRuntimeSizeExpandedField from './ServingRuntimeSizeExpandedField';
+import useGPUSetting from '../../../../notebookController/screens/server/useGPUSetting';
+import { GpuSettingString } from '../../../../../types';
 
 type ServingRuntimeSizeSectionProps = {
   data: CreatingServingRuntimeObject;
   setData: UpdateObjectAtPropAndValue<CreatingServingRuntimeObject>;
   sizes: ServingRuntimeSize[];
+  gpuSetting: GpuSettingString;
 };
 
 const ServingRuntimeSizeSection: React.FC<ServingRuntimeSizeSectionProps> = ({
   data,
   setData,
   sizes,
+  gpuSetting,
 }) => {
   const [sizeDropdownOpen, setSizeDropdownOpen] = React.useState(false);
+  const { available: gpuAvailable, count: gpuCount } = useGPUSetting(gpuSetting || 'hidden');
 
-  // Leaving this to enable GPU in next release
-  // const onChangeGPU = (event: React.FormEvent<HTMLInputElement>) => {
-  //   const target = event.target as HTMLInputElement;
-  //   setData('gpus', parseInt(target.value));
-  // };
+  const onChangeGPU = (event: React.FormEvent<HTMLInputElement>) => {
+    const target = event.target as HTMLInputElement;
+    setData('gpus', parseInt(target.value));
```

Comment:
If the field starts with a non-number, you'll get NaN.

Relevant quote from MDN:
> If the first character cannot be converted to a number [...], parseInt returns NaN. Leading whitespace is allowed.

Docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#description

It should be noted that NumberInputs are not immune from showcasing a non number.